### DISCUSSION TITLE
Pagespeed Insights Lab data

### DISCUSSION THREAD

**Discussion by vimal-here** [GoogleChrome/lighthouse#12894] [Q&A]
**Summary**

Read from some articles that Pagespeed Insights uses lab data of last 28 days to calculate Lighthouse performance scores. And this data collections happen from only 4 locations geographically.

My website is for UAE and Lighthouse does not have servers in Middle East. 
Q1 - Is that one of the reason why there is a big variance in the score for Chrome Dev Tools score vs PSI?

My website has **79+ score** in Chrome Web Tools. But Pagespeed Insights give only **19** (Desktop)

Q2 - Can we tell business that we can consider the Chrome Dev Tools Lighthouse Score instead of PSI? Because the PSI scores are fluctuating.

**Comment by connorjclark** [comment]
PSI likely handles this via a server in Belgium.

> My website has 79+ score in Chrome Web Tools. But Pagespeed Insights give only 19 (Desktop)

Are you running a desktop or mobile audit in devtools?

>  Can we tell business that we can consider the Chrome Dev Tools Lighthouse Score instead of PSI? Because the PSI scores are fluctuating.

Focus on the Crux data provided by PSI. Use the lab tools internally as developers to guide your way to better Crux scores.
